src/main/web-portal/src/App.tsx

```diff
@@ -1,15 +1,16 @@
 import './App.css';
 import { Var, Val, VM, Session, connect } from './mvvm/backend-binder';
-import LoginView from './starter/LoginView';
+import LoginView from './views/LoginView';
```

Comment:
I like that you renamed the directory to `views`, this not only makes sense, but it is kinda best practise! Nice.